src/cider/nrepl/middleware/debug.clj

```diff
@@ -452,18 +451,18 @@ this map (identified by a key), and will `dissoc` it afterwards."}
   [& body]
   ;; NOTE: *msg* is the message that instrumented the function,
   `(let [~'STATE__ {:msg ~(let [{:keys [code id file line column ns]} *msg*]
-                           (-> {:code code,
+                            (-> {:code code
                                 ;; Passing clojure.lang.Namespace object
                                 ;; as :original-ns breaks nREPL in bewildering
                                 ;; ways.
-                                :original-id id, :original-ns (str (or ns *ns*)),
-                                :file file, :line line, :column column}
+                                 :original-id id, :original-ns (str (or ns *ns*))
```

Comment:
I see here the code and the comments got misaligned.

Reply:
I wonder if this is related to weavejester/cljfmt#111.

Reply:
Yes, this looks like a bug, and I opened weavejester/cljfmt#112. Try editing the comments inside that hash-map to change their indentation any way you like, and `lein cljfmt check` doesn't complain about it nor does `lein cljfmt fix` apply a fix.

For now I'll just make the fix by hand.